### ISSUE TITLE
Define Scenario "Serving Drinks".

### ISSUE THREAD

**Issue by PatrickNa** [b-it-bots/mas_domestic_robotics#106]
We have prepared the scenario storing groceries already. Now it is time to proceed with the next task and set it up in parallel. The scenario suggested in this issue is the "Serving Drinks" scenario.

## Context
The "Serving Drinks" scenario is described in the newest rule book of the RoboCup competition (2019). 

## Expected Behavior
Similar to storing groceries a scenario in mdr_robocup_tasks needs to be defined.

**Comment by henrikschnor** [comment]
Is this issue specifically for creating the scenario setup and the state machine? If not, could we edit this one? I created another issue about serving drinks (see above) where I added some of my thoughts.

**Comment by PatrickNa** [comment]
Yes. It is specific for creating the scenario setup and the state machine.

**Comment by henrikschnor** [comment]
There is a bonus reward available for this task:
> 1.  Preemptively informing a guest that the ordered drink is not available (250pts)

To do this, we need to go to the bar and observe it for available drinks before we take orders. Since we already have code for this, I'd say we try to do this (there is no penalty if we fail).
